code/src/java/pcgen/gui2/PCGenFrame.java

```diff
@@ -125,12 +122,13 @@
  * The main window for PCGen. In addition this class is responsible for providing 
  * global UI functions such as message dialogs. 
  */
-public final class PCGenFrame extends JFrame implements UIDelegate
+public final class PCGenFrame extends JFrame implements UIDelegate, CharacterSelectionListener
 {
 
 	private final PCGenActionMap actionMap;
 	private final CharacterTabs characterTabs;
 	private final PCGenStatusBar statusBar;
+	private final PCGenMenuBar pcGenMenuBar;
```

Comment:
I find it odd that we are adding the menubar here for the first time.  Something seems wrong here.

Reply:
We're just keeping a referencence. The menu bar was always created in this class. See line 180 of the "old" side.

Reply:
Ah I see now thanks